src/main/java/cloud/eppo/cache/ExpiringInMemoryAssignmentCache.java

```diff
@@ -0,0 +1,18 @@
+package cloud.eppo.cache;
+
+import cloud.eppo.api.AbstractAssignmentCache;
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.apache.commons.collections4.map.PassiveExpiringMap;
+
+public class ExpiringInMemoryAssignmentCache extends AbstractAssignmentCache {
+  public ExpiringInMemoryAssignmentCache(int cacheTimeout, TimeUnit timeUnit) {
+    super(Collections.synchronizedMap(new PassiveExpiringMap<>(cacheTimeout, timeUnit)));
+  }
+
+  public ExpiringInMemoryAssignmentCache(
```

Comment:
@aarsilv Should these cache implementations be part of the `api` package since they'll be used downstream? or just part of the cache package since they'll be used in downstream only in the sdk and not in the developer's code.

Reply:
Thinking cache package since they won't be directly used